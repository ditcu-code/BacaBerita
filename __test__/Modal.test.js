import 'react-native';
import React from 'react';
import Modal from '../src/components/modal';
import renderer from 'react-test-renderer';

jest.mock("../src/components/modal");

test('Modal Components Snapshot',()=>{
    const snap = renderer.create(
        <Modal />
    ).toJSON();
expect(snap).toMatchSnapshot();
});


describe('a Text', () => {
    it('should be present', () => {
        let tree = renderer.create(<Modal />
            ).toJSON()
    expect((tree, 'Text')).toBeDefined()
       
    })

})
