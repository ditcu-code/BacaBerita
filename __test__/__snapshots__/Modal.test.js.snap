// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Components Snapshot 1`] = `null`;
